sys/hoon.hoon

```diff
@@ -6409,6 +6409,7 @@
               {$9 p/@ q/nock}                           ::  select arm and fire
               {$10 p/$@(@ {p/@ q/nock}) q/nock}         ::  hint
               {$11 p/nock q/nock}                       ::  grab data from sky
+              {$12 p/{p/@ q/nock} q/nock}               ::  edit
```

Comment:
I thought we intended to switch $12 and $11, so that edit was $11 while scrying became $12; otherwise there's a hole in the nock spec proper since scrying isn't part of nock.